ocs_ci/deployment/deployment.py

```diff
@@ -1716,6 +1718,16 @@ def deploy_ocs(self):
                     update_ntp_compute_nodes()
                 assert ceph_health_check(namespace=self.namespace, tries=60, delay=10)
 
+        # Add Replica-1 setings after deployment complition
+        if config.ENV_DATA.get("enable_non_resilient_pools"):
+            logger.info("Setting non-resilient pools")
+            storage_cluster = StorageCluster(
+                resource_name=config.ENV_DATA["storage_cluster_name"],
+                namespace=config.ENV_DATA["cluster_namespace"],
+            )
+            set_non_resilient_pool(storage_cluster)
+            validate_non_resilient_pool(storage_cluster)
```

Comment:
Add assert  or raise exception to `validate_non_resilient_pool` call.  We should not continue if non resilient pool is not formed.